LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -1193,9 +1193,16 @@ BriefSummary="Massively reduces weapon noise and the soldier's impact on mission
 TinySummary="Reduces mission infiltration time"
 LootTooltip="This weapon upgrade, when installed, will reduce the time the soldier requires to infiltrate during a mission. It also reduces weapon noise during combat."
 
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +5%".
```

Comment:
Please use `+<Ability:CRITINCREASE/>%` in place of the numeric percentages. This will ensure that if we ever change the values of the crit chance bonuses, the localisation will automatically update. Same goes for the +10% below.